fact_helper_file/mime/custom_mime_firmware_containers

```diff
@@ -714,3 +714,13 @@
 # SDN unpacker (artificial)
 0	string \x53\x44\x4E	FKIE SDN Unpacker
 !:mime	firmware/sdn
+
+# Draytek Vigor Firmware
+# Source: https://github.com/q3k/vraytekdigor/blob/master/build/fwtool/fwimage.go
+0       string  2RDH    Draytek Vigor Firmware Container
```

Comment:
What about the samples we saw from other draytek devices? Is `2RDH` really a universal magic word for all kind of draytek vigor firmware containers?

Reply:
For now this mime type only covers the Draytek Vigor 167 modem. Specified description in c4c6341d9cfc83e9b4e61e887ec3c2cb98aa8dc4